src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.SafeCreateFile.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal static readonly IntPtr INVALID_HANDLE_VALUE = new IntPtr(-1);  // WinBase.h
+
+        /// <summary>
+        /// Does not allow access to non-file devices. This disallows DOS devices like "con:", "com1:",
+        /// "lpt1:", etc.  Use this to avoid security problems, like allowing a web client asking a server
+        /// for "http://server/com1.aspx" and then causing a worker process to hang.
+        /// </summary>
+        internal static SafeFileHandle SafeCreateFile(
```

Comment:
I do not think that this SafeCreateFile, UnsafeCreateFile, etc. make sense. I would keep the CoreRT version